com.ibm.wala.cast.python.ml.test/source/com/ibm/wala/cast/python/ml/test/TestTensorflowModel.java

```diff
@@ -202,8 +202,24 @@ public void testTf2()
         0); // NOTE: Change to testTf2("tf2_test_dataset.py", "add", 2, 3, 2, 3) once
     // https://github.com/wala/ML/issues/89 is fixed.
     testTf2("tf2_test_tensor_list.py", "add", 2, 3, 2, 3);
-    testTf2("tf2_test_tensor_list2.py", "add", 0, 2);
-    testTf2("tf2_test_tensor_list3.py", "add", 0, 2);
+    //    testTf2("tf2_test_tensor_list2.py", "add", 0, 2);
```

Comment:
Not sure why adding test entails removing others?

Reply:
Addressed in 097a13a7fb12d2a41780769088e32441e886c2f4